<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>AJ Fite and Sara Kipps</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>723</width>
    <height>747</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Solar Irradiance Microforecasting Control Panel</string>
  </property>
  <property name="windowIcon">
   <iconset theme="h">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Solar Irradiance Microforecasting Control Panel</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="3">
     <layout class="QGridLayout" name="gridLayout">
      <item row="8" column="0">
       <widget class="QPushButton" name="stopCapButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Stop Capture</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QGraphicsView" name="graphicsView_7"/>
      </item>
      <item row="1" column="1">
       <widget class="QGraphicsView" name="graphicsView_5"/>
      </item>
      <item row="3" column="1">
       <widget class="QGraphicsView" name="graphicsView_8"/>
      </item>
      <item row="2" column="1">
       <widget class="QGraphicsView" name="graphicsView_4"/>
      </item>
      <item row="2" column="0">
       <widget class="QGraphicsView" name="graphicsView_2"/>
      </item>
      <item row="1" column="2">
       <widget class="QGraphicsView" name="graphicsView_6"/>
      </item>
      <item row="3" column="0">
       <widget class="QGraphicsView" name="graphicsView_3"/>
      </item>
      <item row="1" column="0">
       <widget class="QGraphicsView" name="graphicsView"/>
      </item>
      <item row="2" column="2">
       <widget class="QGraphicsView" name="graphicsView_9"/>
      </item>
      <item row="6" column="1">
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QLCDNumber" name="solarIrradiance">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="3">
       <widget class="QPlainTextEdit" name="consoleWidget">
        <property name="undoRedoEnabled">
         <bool>false</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QPushButton" name="startCapButton">
        <property name="text">
         <string>Start Capture</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Capture Progress</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Solar Irradiance</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Camera Count</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLCDNumber" name="cameraCount"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>723</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <addaction name="actionCapture_Settings"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Abo&amp;ut</string>
    </property>
    <addaction name="actionSource_Code"/>
    <addaction name="actionLicense"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuAbout"/>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCapture_Settings">
   <property name="text">
    <string>&amp;Capture Settings</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionSource_Code">
   <property name="text">
    <string>&amp;Source Code</string>
   </property>
  </action>
  <action name="actionLicense">
   <property name="text">
    <string>&amp;License</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
